Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -225,7 +225,7 @@
 "Keep_up_with_future_live_streams" = "Keep up with future live streams";
 "Kickstarter_Music" = "Kickstarter Music";
 "Kickstarter_if_funded" = "Kickstarter (if funded)";
-"Kickstarter_is_not_a_store" = "Kickstarter is not a store.";
```

Comment:
So before this string makes it back through Crowdin we should temporarily use this when setting it on the `DeprecatedPledgeVC`

```
localizedString(
  key: "Kickstarter_is_not_a_store", 
  defaultValue: "Kickstarter is not a store."
)
```

Unfortunately we will lose German, French and other translations but since it's only temporary it should be fine...I would also create a follow up card in Trello (i.e. `Localize Kickstarter_is_not_a_store`) so that we won't forget about this